cross/libbluray/Makefile

```diff
@@ -0,0 +1,19 @@
+PKG_NAME = libbluray
+PKG_VERS = 0.9.3
+PKG_EXT = tar.bz2
+PKG_DIST_NAME = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
+PKG_DIST_SITE = http://ftp.videolan.org/pub/videolan/libbluray/$(PKG_VERS)
+PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
+
+DEPENDS = cross/libxml2 cross/freetype cross/fontconfig
+
+HOMEPAGE = http://www.videolan.org/developers/libbluray.html
+COMMENT  = libbluray is an open-source library designed for Blu-Ray Discs playback for media players.
+LICENSE  = LGPL
+
+GNU_CONFIGURE = 1
+
+CONFIGURE_ARGS = --disable-examples --disable-doxygen-doc --disable-doxygen-dot
```

Comment:
This doesn't build out of the box. Either set `--disable-bdjava`, or ensure that the requirements are met (Apache Ant, Java JDK)